crates/bevy_gizmos/src/arrows.rs

```diff
@@ -0,0 +1,106 @@
+//! Additional Gizmo Functions -- Arrows
+
+use crate::prelude::Gizmos;
+use bevy_math::{Quat, Vec2, Vec3};
+use bevy_render::color::Color;
+
+pub struct ArrowBuilder<'a, 's> {
+    gizmos: &'a mut Gizmos<'s>,
+    start: Vec3,
+    end: Vec3,
+    color: Color,
+    tip_length: f32,
+}
+
+/// A builder returned by [`Gizmos::arrow`] and [`Gizmos::arrow_2d`]
+impl ArrowBuilder<'_, '_> {
+    /// Change the length of the tips to be `length`.
+    /// The default tip length is [length of the arrow]/10, which is sufficient for most applications.
+    ///
+    /// # example
```

Comment:
```suggestion
    /// # Example
```

Reply:
done x3 in [dba55c8](https://github.com/bevyengine/bevy/pull/10550/commits/dba55c8144fc0fc52088cb5a16177f9e8c7f458f)